deps_licenses/licenses_mac_intel_user.txt

```diff
@@ -41,7 +42,7 @@ protobuf, 3.20.3, BSD-3-Clause
 psutil, 5.9.5, BSD License
 pycparser, 2.21, BSD License
 pydantic, 1.10.12, MIT License
-pytest, 7.2.0, MIT License
+pytest, 7.4.1, MIT License
```

Comment:
by the way, I updated Mac intel license files. Would there be a way to do it automatically in your action, @RomanBredehoft ?

Reply:
not sure I understand the question ? The action (which was made by @fd0r , credits to him !) updates automatically linux and mac intel licenses once triggered) , but you've already done it for mac here (are you talking about linux then ?)

Reply:
and ok, the question is for @fd0r then, sorry for mis direction

Reply:
OK I see: the action to refresh is not launched automatically, is it? and, as we check licenses only on linux in our builds, we don't see it's not uptodate. But at the same time, when the last person launched the refresh for linux, the refresh should also have been done for Mac intel, no? 

Let's wait @fd0r maybe

Reply:
no we have to trigger it by targeting our branch, not sure we want to run a make license and add a commit each time we push something (if it's even doable) ..

if you trigger the action, it will do both linux and mac intel yes. You can try it out, it's just here : https://github.com/zama-ai/concrete-ml/blob/main/.github/workflows/update_licenses.yaml

The only thing @fd0r wanted to improve is that for now, it will open 2 PRs targetting your branch (no need to wait for CIs as they don't target main) that you'll need to merge, but ultimately we'll make them directly commit in your branch (as notebook refreshed are done)

Reply:
>  not sure we want to run a make license and add a commit each time we push something

we would only commit if something changed. But ok keep it the way it is, if it's convenient for you

Reply:
I'm still not sure we can commit within a ci without having the recursive error you faced long ago anyway !